app/questionnaire/navigator.py

```diff
@@ -96,13 +97,13 @@ class Navigator:
     PRECEEDING_INTERSTITIAL_PATH = ['introduction']
     CLOSING_INTERSTITIAL_PATH = ['summary', 'thank-you']
 
-    def __init__(self, survey_json, metadata=None, answer_store=None):
+    def __init__(self, survey_json, metadata=None, answer_store=None, group_id=None, group_instance=0):
```

Comment:
I don't think initialising with group_id/group_instance makes sense here. I'd prefer that we limit location args to the methods we call against navigator. In this case it looks as if we could  do so by adding these params to get_routing_path method and keeping the init as is?